pkg/artifactory/resource/replication/resource_artifactory_push_replication.go

```diff
@@ -55,23 +54,6 @@ type UpdatePushReplication struct {
 	Replications           []updateReplicationBody `json:"replications,omitempty"`
 }
 
-var pushReplicationSchemaCommon = map[string]*schema.Schema{
```

Comment:
Why get rid of this?

Reply:
We already have this schema with the name `replicationSchemaCommon` in `resource_artifactory_replication_config.go`, since it's the same, I just reuse it and remove the duplicate.